src/app/product.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProductService {
+  private url = 'https://morgantiluxury.com/wp-json/wc/v3/';
+  private consumerKey = 'ck_ec268086198dd321148000cc8c28a35f85525c00';
+  private consumerSecret = 'cs_fcd73d02e6d0d53249ba5017b8d51c4e95dac8b6';
```

Comment:
we already have these keys in the environment files, import it from there only

Reply:
Done @farman8799